util/src/main/java/tc/oc/pgm/util/parser/StringReader.java

```diff
@@ -89,6 +91,27 @@ public boolean pollIf(char ch) {
     return false;
   }
 
+  /**
+   * Peeks at the next char ignoring whitespace, if it's one of {@param chs}, polls it and returns
+   * the matching character
+   *
+   * @param chs The characters to expect next
+   * @return one of {@param chs} if it is the next char, and it was skipped. -1 otherwise.
+   */
+  public int pollFor(char... chs) {
```

Comment:
atp i'd rather have a pollQuote that is hardcoded to poll for " or ' instead of an arbitrary array of chars, and returns a char instead of an int, which may be `"`, `'` or a NULL char (ie: char 0) if none found